src/classification/zoneOfPartialPreservation/motorZPP.ts

```diff
@@ -404,38 +403,30 @@ export function checkLevel(state: State): Step {
 export function getTopAndBottomLevelsForCheck(state: State): Step {
   const motorLevels = state.motorLevel.replace(/\*/g, '').split(',');
   const top = motorLevels[0] as SensoryLevel;
-
-  // We exclude not normal T1 values as there would be no propagation for that case
-  const includeThoracicAndLumbarSensoryLevels = /T1\*?(,|$)/.test(state.motorLevel) && /^(5|NT|(NT|[0-4])\*\*)$/.test(state.side.motor['T1']);

```

Comment:
This is the rule we are removing. It fails to predict cases with combinations of normal values and `NT` values between `T1` and `L2`. Trying to make it work makes the algorithm more complicated. It is better to always check all sensory values. They are not that many.